doc/user/sql/create-index.md

```diff
@@ -68,45 +68,33 @@ _col&lowbar;ref_**...** | The columns to use as the key into the index, listed i
     consider creating another materialized view that uses `SELECT some_subset
     FROM this_view...`.
 
-### Column order
-
-The order of indexed columns does impact the underlying structure of the index,
-i.e. columns are indexed in the order in which you list them.
-
-This can be most clearly seen when joining two relations; in these cases, you
-want the relations to have join keys listed in the same order to ensure simple
-point lookups (vs. index scans).
+### Structure
 
-For example, if we are joining relations `a` and `b` on columns `order_id` and
-`customer_id`, the following indexes would perform well:
+Indexes in Materialize have the following structure for each unique row.
 
--    ```sql
-     CREATE INDEX a_o_idx ON a(order_id, customer_id);
-     CREATE INDEX b_o_idx ON b(order_id, customer_id);
-     ```
+```nofmt
+((tuple of indexed columns), (tuple of the row, i.e. stored columns))
+```
 
--   ```sql
-    CREATE INDEX a_c_idx ON a(customer_id, order_id);
-    CREATE INDEX b_c_idx ON b(customer_id, order_id);
-    ```
+Index are maintained in ascending order for the tuple of indexed columns.
 
-However, transposing the order of the columns would result in **poor**
-performance:
+#### Indexed columns vs. stored columns
 
--    ```sql
-     CREATE INDEX a_t_idx ON a(order_id, customer_id);
-     CREATE INDEX b_t_idx ON b(customer_id, order_id);
-     ```
+Automatically created columns index all of a table's columns, unless Materialize
+is provided or can infer a unique key for the result set.
 
-### Structure
+For instance, unique keys can be...
```

Comment:
Is this a TODO? I can help with the text, but wanted to make sure it doesn't sneak past us! :D

Reply:
Wait, I'm sorry. I understand now. My brain didn't parse the surrounding markdown in the way it would present when rendered.